kubernetes/services/concent-api.yml.j2

```
@@ -14,6 +14,10 @@ spec:
         - name:             concent-api
           image:            {{ docker_registry }}/{{image_prefix }}concent-api:{{ image_version }}
           imagePullPolicy:  Always
+          command:             [
+              "/usr/local/bin/run-gunicorn.sh",
+              "{{ concent_versions[concent_version] | chained_get('amount_of_gunicorn_workers.concent_api') | default('{{ amount_of_gunicorn_workers.concent_api }}', true) }}"
```

Why pass `amount_of_gunicorn_workers.concent_api` through `{{}}` and the put it in a string? Why not use it directly?

```suggestion
              "{{ concent_versions[concent_version] | chained_get('amount_of_gunicorn_workers.concent_api') | default(amount_of_gunicorn_workers.concent_api, true) }}"
```

---

Ok. I correct this in all playbooks.